lib/model/schedule_candidate/schedule_candidate.dart

```diff
@@ -9,8 +9,8 @@ part 'schedule_candidate.g.dart';
 abstract class ScheduleCandidate with _$ScheduleCandidate {
   const factory ScheduleCandidate({
     @required String id,
-    //@required TimeStamp start,
-    // @required TimeStamp end,
+    @required DateTime start,
+     @required DateTime end,
```

Comment:
```suggestion
    @required DateTime end,
```
インデント が若干ズレとる？w(細かっ🙏)